website/src/content/docs/docs/howtos/Generate with tsp-client/local_spec_quickstart.md

```diff
@@ -0,0 +1,42 @@
+---
+title:  Quickstart - Local development
```

Comment:
Wondering if this name makes sense right next to `tsp-client usage`

should we name them differently?

Reply:
I agree...Maybe it should be:
- `tsp-client command documentation`
- `tsp-client quickstart`

OR maybe just remove tsp-client and do:
- `Supported commands`
- `Quickstart (local development)`

Reply:
I liked the second option more so updated to that, let me know if you have a better idea!